File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntAutomationStepSendEmail.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdField;
import com.neome.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioEntAutomationStepSendEmail extends StudioEntAutomationStepSendMessageBase
{
  @Nullable
  public MetaIdField[] bccFieldIdSet;

  @Nullable
  public MetaIdVar bccSetOfUserVarId;

  @Nullable
  public MetaIdField[] ccFieldIdSet;

  @Nullable
  public MetaIdVar ccSetOfUserVarId;

  @Nullable
  public MetaIdField mediaFieldId;

  @Nullable
  public StudioValueVarIdParagraph messageVarId;

  @Nullable
  public MetaIdField replyToFieldId;

  @Nullable
  public MetaIdVar replyToSetOfUserVarId;

  @Nullable
  public StudioValueVarIdText subjectVarId;

  @Nullable
  public MetaIdField[] toFieldIdSet;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntAutomationStepSendEmail.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdVar

interface StudioEntAutomationStepSendEmail : StudioEntAutomationStepSendMessageBase {
    val bccFieldIdSet: List<MetaIdField>?
    val bccSetOfUserVarId: MetaIdVar?
    val ccFieldIdSet: List<MetaIdField>?
    val ccSetOfUserVarId: MetaIdVar?
    val mediaFieldId: MetaIdField?
    val messageVarId: StudioValueVarIdParagraph?
    val replyToFieldId: MetaIdField?
    val replyToSetOfUserVarId: MetaIdVar?
    val subjectVarId: StudioValueVarIdText?
    val toFieldIdSet: List<MetaIdField>?
}
